charts/secrets-store-csi-driver/templates/_helpers.tpl

```diff
@@ -12,13 +12,17 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 If release name contains chart name it will be used as a full name.
 */}}
 {{- define "sscd.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
```

Comment:
Is this the result of running `make promote-staging-manifest`?

Reply:
yeah, this change was added in https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/671

Reply:
ah i was forgetting the difference between the charts vs staging folders again